packages/sdk-js/test/json.test.ts

```diff
@@ -99,11 +99,11 @@ describe("json test suite", () => {
 
   it.each((cases as Cases).evalCondition)(
     "evalCondition[%#] %s",
-    (name, condition, value, expected) => {
+    (name, condition, value, expected, idLists = {}) => {
```

Comment:
When we change `cases.json`, we need to change a few other things at the same time:

1. Update the `Cases` type above in this file
2. Bump the spec version at the top of the `cases.json` file
3. On `build-your-own.mdx`, bump the spec version at the top and add it to the changelog at the bottom
4. On `build-your-own.mdx`, update the cases.json documentation at the bottom where it lists out all inputs/outputs